bylaws/008-psr-evolution.md

```diff
@@ -28,11 +28,11 @@ Since releasing new versions of the interfaces MUST NOT alter the PSR in its beh
 
 ### Practical example
 
-A common case for an upgrade in the interfaces is to add parameters and return types, since they are a new language feature introduced by PHP 7, and many PSR interfaces predates that release. In the next paragraph, we'll try to use PSR-3 as an example to show how a PSR interface should be updated.
+A common case for an upgrade in the interfaces is to add typehints for parameters and returns, since they are new language features introduced by PHP 7, and many PSR interfaces predate that release. We'll use a method from PSR-3 as an example of how a PSR interface could be updated.
```

Comment:
Those are not "hints", but full-blown types...
```suggestion
A common case for an upgrade in the interfaces is to add types for parameters and returns, since they are new language features introduced by PHP 7, and many PSR interfaces predate that release. We'll use a method from PSR-3 as an example of how a PSR interface could be updated.
```

Reply:
https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#52-author
5.14. @throws is that correct?  if I have catched ,should I still comment on it ?